index.bs

```diff
@@ -1758,7 +1813,10 @@ To <dfn noexport>parse source-registration JSON</dfn> given a [=byte sequence=]
 1. If |value|["`debug_reporting`"] [=map/exists=] and is a [=boolean=], set
     |debugReportingEnabled| to |value|["`debug_reporting`"].
 
-1. Let |eventReportWindows| be the result of [=obtaining effective windows=] given |sourceType|, |sourceTime|, and |eventReportWindow|.
+1. Let |eventReportWindows| be the result of [=obtaining default effective windows=] given |sourceType|, |sourceTime|, and |eventReportWindow|.
+1. If |value|["`event_report_window`"] does not [=map/exist=]:
+    1. Set |eventReportWindows| to the result of [=parsing report windows=] with |value|, |sourceTime|, and |eventReportWindows|.
```

Comment:
Should it be an error for `event_report_window` and `event_report_windows` to be set in the same registration?

Alternatively, should we reuse the `event_report_window` name and allow it to be either a single value as today *or* the flex lite stuff added in this PR?

Reply:
Using the name `event_report_window` for multiple windows feels very unintuitive to me so I'd prefer making this an error.

Reply:
Agreed on the unintuitiveness, but we did something similar for multi-destination. I don't feel strongly either way, but agree that it should be an error if we do not reuse.

Reply:
Using `destination` to refer to a set of destinations feels less egregious to me than `window` referring to a set of windows. I think this might be because "destination" is a more abstract term.